javascript-forms/form-values.js

```diff
@@ -0,0 +1,11 @@
+const $form = document.querySelector('form');
+const input = {};
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  input['name:'] = $form.elements.name.value;
+  input['email:'] = $form.elements.email.value;
+  input['message:'] = $form.elements.message.value;
```

Comment:
You're adding an extra colon by doing this. If you check out the console log, the object is showing `name::`. Either remove the colon, or use dot notation